config/src/main/java/com/typesafe/config/impl/ConfigBitInteger.java

```diff
@@ -0,0 +1,62 @@
+/**
+ *   Copyright (C) 2011-2012 Typesafe Inc. <http://typesafe.com>
+ */
+package com.typesafe.config.impl;
+
+import java.io.ObjectStreamException;
+import java.io.Serializable;
+import java.math.BigInteger;
+
+import com.typesafe.config.ConfigOrigin;
+import com.typesafe.config.ConfigValueType;
+
+final class ConfigBigInteger extends ConfigNumber implements Serializable {
+
+    private static final long serialVersionUID = 2L;
+
+    final private BigInteger value;
+
+    ConfigBigInteger(ConfigOrigin origin, BigInteger value, String originalText) {
+        super(origin, originalText);
+        this.value = value;
+    }
+
+    @Override
+    public ConfigValueType valueType() {
+        return ConfigValueType.NUMBER;
+    }
+
+    @Override
+    public BigInteger unwrapped() {
+        return value;
+    }
+
+    @Override
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://www.codacy.com/app/hp/config/file/2444185959/issues/source?bid=3245965&fileBranchId=3245975#l34)